cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzurePlatformResources.java

```diff
@@ -343,6 +346,16 @@ public CloudResourceGroups resourceGroups(ExtendedCloudCredential cloudCredentia
         return new CloudResourceGroups(resourceGroups);
     }
 
+    @Override
+    public CloudPrivateDnsZones privateDnsZones(ExtendedCloudCredential cloudCredential, Region region, Map<String, String> filters) {
+        PagedList<PrivateZone> azurePrivateDnsZones = azureClientService.getClient(cloudCredential).getPrivateDnsZoneList();
```

Comment:
is it verified that the client's call lists zones across subscriptions, too?
I am asking because of this: 
`privatednsManager authenticate(RestClient restClient, String subscriptionId)`

Reply:
I now tested and indeed subscriptionId needs to be passed to the privateDnsZoneManager. Thank you very much for that!

Reply:
I modified the code that it:
- retrieves all subscriptions
- gathers the list of all privateDnsZones from the available subscriptions
I checked and it returned also the zones from another subscription.
Luckily there is no need to ask for more permissions as the "Microsoft.Resources/subscriptions/read" permission is already in the minimum role definition, published on Cloudera docs (https://docs.cloudera.com/cdp/latest/requirements-azure/topics/mc-azure-credential.html#mc-azure-permissions__section_tgf_2td_x4b)